File: python/1000-7.py
from time import sleep

def main():
	try :
		n = 1000
		while n > 0:
			if n > 13:
				n -= 7
				sleep (0.08)
				print(f'{n} - 7 = {n - 7}')
			if n == 13:
				print('Я гуль!')
				break
	except:
		print('Ошибка')

if __name__ == '__main__':
	main()

File: zig/src/1000-7.zig
const std = @import("std");

pub fn main() !void {
    var i: i32 = 1000;
    while (i > 7) {
        std.debug.print("{} - 7 = {}\n", .{ i, i - 7 });
        i -= 7;
        std.time.sleep(50 * 1_000_000);
    }
    std.debug.print("Я гуль\n", .{});
}
